test/mysql.autoupdate.test.js

```diff
@@ -222,6 +220,56 @@ describe('MySQL connector', function() {
     verifyMysqlColumnNameAutoupdate(done);
   });
 
+  it('should update the nullable property of "first_name" to false', function(done) {
+    // update the model "required" property
+    updateSchemaColRenameTest();
```

Comment:
Instead of creating two functions and calling them inside the test, could you move the logic on the function in the test case itself?